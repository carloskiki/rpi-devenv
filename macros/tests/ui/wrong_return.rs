#[macros::main]
const fn kernel_main() -> i32 {
    0
}

fn main() {}
