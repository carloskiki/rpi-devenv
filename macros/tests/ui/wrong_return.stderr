error: main function return type must be `!`
 --> tests/ui/wrong_return.rs:2:27
  |
2 | const fn kernel_main() -> i32 {
  |                           ^^^
